team/templates/team/organizer_partial.html

```diff
@@ -21,6 +21,9 @@ <h2>{{ organizer.name }}</h2>
     {% if organizer.twitter %}
       <a href="https://www.twitter.com/{{ organizer.twitter }}" target="_blank"><i class="fa-brands fa-2x fa-twitter"></i></a>
     {% endif %}
+    {% if organizer.mastodon %}
+      <a href="https://{{ organizer.mastodon }}" target="_blank"><i class="fa-brands fa-2x fa-mastodon"></i></a>
```

Comment:
```suggestion
      <a href="https://{{ organizer.mastodon | remove_scheme_prefix }}" target="_blank"><i class="fa-brands fa-2x fa-mastodon"></i></a>
```
<img width="309" alt="image" src="https://user-images.githubusercontent.com/5224106/206655530-9202c2b9-f2a7-4b45-bb92-6cc71f8e0934.png">

**before:**
<img width="414" alt="image" src="https://user-images.githubusercontent.com/5224106/206655473-c745050c-2583-46f4-896b-160fad6d9ba2.png">

**after:**
<img width="389" alt="image" src="https://user-images.githubusercontent.com/5224106/206656422-81b84fb9-3190-4507-84fa-d903d8fdae5d.png">

also importing the filter (not sure what Django best practice is here):

```
{% load social_tags %}
```

Reply:
Ahh, good call!